src/Resources/js/modules/combined-meal-dialog.ts

```diff
@@ -0,0 +1,129 @@
+import 'jquery-ui/ui/widgets/dialog';
+
+export class CombinedMealDialog {
+    readonly containerID: string = '#combined-meal-selector';
+    title: string;
+    path: string;
+    opts: CombinedMealDialogOptions;
+
+    $form: JQuery;
+    $dialog: JQuery;
+
+    constructor(private dishes: Dish[], path: string, opts: CombinedMealDialogOptions) {
+        this.title = this.getTitle(dishes);
+        this.path = path;
+        this.$form = this.buildForm(dishes);
+        this.opts = opts;
+    }
+
+    open() {
+        this.$dialog = $(this.containerID).empty().append(this.$form).dialog({
+            modal: true,
+            width: 400,
+            maxWidth: 500,
+            title: this.title,
+            draggable: false,
+            buttons: {
+                'OK': this.handleOk.bind(this),
+                'Cancel': this.handleCancel.bind(this)
+            },
+            create: this.handleCreate
+        });
+    }
+
+    getTitle(dishes: Dish[]): string {
+        return dishes.reduce(function(title: string, dish: Dish) {
+            return ('' === title) ? dish.title : `${title} & ${dish.title}`;
+        }, '');
+    }
+
+    buildForm(dishes: Dish[]): JQuery {
+        let $form = $('<form method="post"></form>');
+        let $formFields = this.getFormFields(dishes);
+        $form.prepend($formFields);
+
+        return $form;
+    }
+
+    getFormFields(dishes: Dish[]): JQuery {
+        let $dishes = $('<div class="dishes"></div>');
+
+        dishes.forEach((dish, index) => {
+            let $dishField = this.getDishField(dish, index);
+            $dishes.append($dishField);
+        });
+
+        return $dishes;
+    }
+
+    getDishField(d: Dish, index: number): JQuery {
+        const fieldName = `dishes[${index}]`;
+
+        if (0 === d.variations.length) {
+            return this.getRadioButton(fieldName, d.slug, true, d.title, {wrapperClass: 'dish'});
+        }
+
+        let $dish = $(`<div class="dish"><div class="title">${d.title}</div></div>`);
```

Comment:
in diesem fall würde ich tatsächlich wieder den "normalen" string zusammenbau verwenden ala '+ d.title +'

Reply:
$$$$ 
Done.